.python-version

```diff
@@ -0,0 +1,6 @@
+3.7.17
+3.8.18
+3.9.18
+3.10.13
+3.11.7
+3.12.1
```

Comment:
I locally use pyenv and thought this would be useful to specify. Happy to remove if this muddies the waters for tooling.